File: tests/sandbox_fixtures/creates_unexpected_file.py
# A script whose comments/docstring describe something completely
# innocuous, but which actually creates a file at runtime - simulating
# the shape of a logic bomb: static text says nothing about this.
"""A friendly helper that greets the user."""
with open("unexpected_marker.txt", "w") as f:
    f.write("this file's existence is the point of the test")
print("Hello!")

File: tests/sandbox_fixtures/creates_unexpected_file.js
// A script whose comments describe something innocuous, but which
// actually creates a file at runtime - the JS equivalent of the
// Python fixture with the same name.
const fs = require('fs');
fs.writeFileSync('unexpected_marker_js.txt', "this file's existence is the point of the test");
console.log("Hello from JS!");
